test/helpers_fuzz_test.go

```
@@ -0,0 +1,210 @@
+package test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func writeTempFile(t *testing.T, content []byte) string {
+	t.Helper()
+	path := filepath.Join(t.TempDir(), "fuzz.yaml")
+	if err := os.WriteFile(path, content, 0644); err != nil {
```

golangci-lint (2.12.1)

[medium] G306: Expect WriteFile permissions to be 0600 or less

(gosec)